test/jdk/java/net/URLPermission/URLPermissionNonASCIITest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Ryan, there's a comma missing here after `2022`.

Reply:
File has been deleted and test has been added to URLPermissionTest class